taimen/config.json

```diff
@@ -0,0 +1,307 @@
+{
+  "device-name": "pixel2",
```

Comment:
Better rename this to pixel2-xl to avoid confusion when reading config (not used yet by the scripts).